commit cd07072158b104f2e30a14bf5162095bb3b9c230
Author: P33M <p33m@github.com>
Date:   Wed May 1 15:00:05 2019 +0100

    dts: Increase default coherent pool size
    
    dwc_otg allocates DMA-coherent buffers in atomic context for misaligned
    transfer buffers. The pool that these allocations come from is set up
    at boot-time but can be overridden by a commandline parameter -
    increase this for now to prevent failures seen on 4.19 with multiple
    USB Ethernet devices.
    
    see: https://github.com/raspberrypi/linux/issues/2924

diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index 55a03c0d5e1c..ecdb36e69c87 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -3,7 +3,7 @@
 
 / {
 	chosen {
-		bootargs = "";
+		bootargs = "coherent_pool=1M";
 		/delete-property/ stdout-path;
 	};
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1482_cd07072158b104f2e30a14bf5162095bb3b9c230.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1482_cd07072158b104f2e30a14bf5162095bb3b9c230.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
